README.md

```diff
@@ -9,7 +9,7 @@ zkRust supports generating proofs for executable scripts. Specifically, zkRust s
 First make sure [Rust](https://www.rust-lang.org/tools/install) is installed on your machine. Then install the zkVM toolchains from [risc0](https://github.com/risc0/risc0) and [sp1](https://github.com/succinctlabs/sp1) by running:
 
 ```sh
-make install
+curl -L https://raw.githubusercontent.com/yetanotherco/zkRust/main/install_zkrust.sh | bash
```

Comment:
this should not be replaced, we need a guide for local development. Keep the `make install` and make another section for installing directly

Reply:
also, the downloaded binary does not work without the toolchains, so we may have to give guidelines to install them or add their installation to the script

Reply:
Addressed!